cli/src/command.rs

```diff
@@ -150,6 +152,17 @@ pub fn run() -> Result<()> {
 						grandpa_pause,
 					).map(|(components, _, _)| components)
 				})
+			} else if chain_spec.is_test() {
+				runtime.run_node_until_exit(|config| match config.role {
+					Role::Light => unimplemented!(),
+					_ => polkadot_test_service::polkadot_test_new_full(
+							config,
+							None,
+							None,
+							authority_discovery_enabled,
+							6000,
+						).map(|(components, _, _, _, _)| components)
+				})
```

Comment:
@bkchr I added this. It's useful if you want to run from the command-line. I can revert it if you say we shouldn't expose that

Reply:
The point was that we don't need to compile the `test-runtime` when doing a normal build. So, we don't need it.

Reply:
done